modules/howtos/examples/n1ql_ops.py

```diff
@@ -0,0 +1,152 @@
+from couchbase_core.mutation_state import  MutationState
+from couchbase.cluster import Cluster
+cluster=Cluster("localhost")
+bucket=cluster.bucket("default")
+collection=bucket.default_collection()
+"""= N1QL Queries from the SDK
+:navtitle: N1QL from the SDK
+:page-topic-type: howto
+:page-aliases: n1ql-query
+
+[abstract]
+You can query for documents in Couchbase using the N1QL query language, a language based on SQL, but designed for structured and flexible JSON documents.
+Querying can solve typical programming tasks such as finding a user profile by email address, facebook login, or user ID.
+
+include::partial$alpha-warning.adoc[]
+
+Our query service uses N1QL, which will be fairly familiar to anyone who's used any dialect of SQL.
+xref:#additional-resources[Further resources] for learning about N1QL are listed at the bottom of the page.
+Before you get started you may wish to checkout the xref:6.5@server:n1ql:n1ql-language-reference/index.adoc[N1QL intro page], or just dive in with a query against our travel sample data set.
+In this case, the one thing that you need to know is that in order to make a Bucket queryable, it must have at least one index defined.
```

Comment:
Reword to: 
In this case, note that before you can query a bucket, you must define at least one index.